controllers/case_studies/delete/post.js

```diff
@@ -0,0 +1,13 @@
+
+const CaseStudy = require('../../../models/caseStudy/casestudy');
+
+module.exports = (req, res) => {
+  CaseStudy.deleteCaseStudy(req.body.company_name, (err, caseStudy) => {
```

Comment:
Şurası hata veriri, req.body'i undefined atarsa eğer karşı taraf throw error olur çöker worker. Bunu engellemek için ya en başa bi if at,
if (!req.body) req.body = {}; // Şeklinde
ya da req.body ? req.body.company_name : null, yapabilirsin orayı. Ama bence ne iyi çözüm direk req.body'i argüman olarak verip delete'te bakmak. Ayrıca company_name ile silemezsin bir şeyi, bunu konuşmuştuk _id'yi kullan. Eğer niye hatırlamıyorsan yaz bana, tekrar anlatayım.